doc/nrf/libraries/others/dult.rst

```diff
@@ -44,6 +44,25 @@ The following Kconfig options are also available for this module:
   * :kconfig:option:`CONFIG_DULT_BT_ANOS_INDICATION_COUNT` - This option allows to configure the number of simultaneously processed GATT indications by the ANOS.
     The default value is set to ``2``.
 
+* :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` - This option enables support for motion detector.
+  The motion detector is an optional feature in the DULT specification.
+  For more details see the `DULT motion detector`_ section of the DULT specification.
+  This option is disabled by default.
+
+  * :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR_TEST_MODE` - This option allows to configure motion detector parameters for testing purposes.
+    These values are defined in the DULT specification and should not be changed in the production code.
+    This option is disabled by default.
```

Comment:
I think it's good to mention what is the default value if those options are not configurable. There's no place for that in your proposition.

Reply:
Because the prompt can be enabled for them so they are a bit different than the kconfigs you mentioned. I think it's beneficial to provide the full information for those kconfigs to let the user see why the test mode is introduced and what it changes.